src/services/CreateDialogService.js

```diff
@@ -1,13 +1,33 @@
 const Dialog = require("../models/Dialog");
 
+const Response = require("../utils/responses");
+
 module.exports = async (req, res) => {
-  const response = await Dialog.create({ ...req.body, user: req.userId })
-    .then((item) => res.status(201).json(item))
-    .catch((error) =>
-      error.name === "ValidationError"
-        ? res.status(400).json({ error: error.message })
-        : res.status(500).json({ error: "Dialog creation failed" })
-    );
-
-  return response;
+  const response = new Response(res);
+
+  const { entities } = response;
+
+  try {
+    const dialog = await Dialog.create({
+      ...req.body,
```

Comment:
I believe that here we have the same problem as Topic 2. Do you agree?

Reply:
really